bucket/grex.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/pemistahl/grex",
+    "description": "A command-line tool for generating regular expressions from user-provided input strings",
```

Comment:
```suggestion
    "description": "Regular expressions generator.",
```

Reply:
Is there a reason for having such short descriptions? I've seen other manifests with informative descriptions as long as mine. The dot at the end of the description does not make sense either as this is not a full grammatical sentence. And it's not used by all manifests anyway.

I've just shortened it a bit now to accommodate you. Is that okay?

Reply:
There is no reason to mention it is command line tool. Inside main bucket there are only command lines

Reply:
I see, then it is okay for me. I've just updated the description again. Is it okay now? Thank you.